docs/source/_static/custom.css

```diff
@@ -0,0 +1,9 @@
+/* 24px margin from readthedocs theme */
+div.nblast {
+    margin-bottom: 19px !important;  /* padding has already 5px */
```

Comment:
> where is .nblast created ?

Ditto, I just checked out the PR, cleaned and built the docs, and opened one of the nbsphinx (example notebook) docs and I didn't see it on the page. `document.querySelector('.nblast')`